docs/user-guide/configuring.md

```diff
@@ -1085,3 +1079,23 @@ foo.js
 ```
 
 This message occurs because ESLint is unsure if you wanted to actually lint the file or not. As the message indicates, you can use `--no-ignore` to omit using the ignore rules.
+
+## Personal Configuration File (deprecated)
+
+⚠️ **This feature has been deprecated**. It will start to print deprecation warnings on ESLint `7.0.0` and will be removed on ESLint `8.0.0`. If you want to continue to use personal configuration file, please use [`--config` CLI option](https://eslint.org/docs/user-guide/command-line-interface#-c---config). If you want to know the discussion of this decision, see [RFC 28](https://github.com/eslint/rfcs/pull/28) and [RFC 32](https://github.com/eslint/rfcs/pull/32).
```

Comment:
```suggestion
⚠️ **This feature has been deprecated**. ESLint will start to print deprecation warnings starting with the `7.0.0` release. These warnings will then be removed in the `8.0.0` release. If you want to continue to use personal configuration files, please use the [`--config` CLI option](https://eslint.org/docs/user-guide/command-line-interface#-c---config). For more information regarding this decision, please see [RFC 28](https://github.com/eslint/rfcs/pull/28) and [RFC 32](https://github.com/eslint/rfcs/pull/32).
```

Reply:
Thank you for your suggestion. I have some questions.

> ESLint will start to print deprecation warnings starting with the `7.0.0` release.

Are not "start to" and "starting with" redundant?

> These warnings will then be removed in the `8.0.0` release. 

What we will remove in `8.0.0` is the personal configuration file functionality. (sure, the warnings will be removed as well, but...). Does it make sense if we clarify what's removed?

Reply:
> Are not "start to" and "starting with" redundant?

Agreed that doesn't flow very well. 

> What we will remove in 8.0.0 is the personal configuration file functionality. (sure, the warnings will be removed as well, but...). Does it make sense if we clarify what's removed?

Ah yeah, I misunderstood that. Given the current sentence structure, it looks like it's implicitly referring to the printing behavior being removed.

Maybe `ESLint will print deprecation warnings beginning with the 7.0.0 release. The feature and deprecation warnings will then be removed in the 8.0.0 release.`?